src/domain/event/eventCard/EventCard.tsx

```diff
@@ -85,79 +85,85 @@ const EventCard: React.FC<Props> = ({ event }) => {
   }, [imageUrl]);
 
   return (
-    <Link
-      aria-label={t('event.eventCard.ariaLabelLink', {
-        name,
-      })}
-      className={classNames(styles.eventCard, {
-        [styles.eventClosed]: eventClosed,
-      })}
-      id={getEventCardId(id)}
-      onClick={handleLinkClick}
-      to={eventUrl}
-    >
-      {/* INFO WRAPPER. Re-order info wrapper and text wrapper on css */}
-      <div className={styles.infoWrapper}>
-        <div className={styles.textWrapper}>
-          <div className={styles.eventName}>
-            <EventName event={event} />
-          </div>
-          <div className={styles.eventDateAndTime}>
-            {!!startTime &&
-              getDateRangeStr({
-                start: startTime,
-                end: endTime,
-                locale,
-                includeWeekday: false,
-                includeTime: true,
-                timeAbbreviation: t('commons.timeAbbreviation'),
-              })}
+    <span data-testid={event.id}>
```

Comment:
span doesn't seem right to me here? Should we use div or add testid to Link

Reply:
I was able to remove it and put it to Link